Tasks/AzureResourceManagerTemplateDeploymentV3/task.json

```diff
@@ -301,6 +301,7 @@
         "ResourceGroupNameNotProvided": "Resource Group name should be provided",
         "LocationNotProvided": "Location is required for deployment",
         "ARMServiceConnectionScope": "ARM Service Conection deployment scope - %s",
-        "CompleteDeploymentModeNotSupported": "Deployment mode 'Complete' is not supported for deployment at '%s' scope"
+        "CompleteDeploymentModeNotSupported": "Deployment mode 'Complete' is not supported for deployment at '%s' scope",
+        "FindMoreDeploymentDetailsAzurePortal": "Find more details about the Azure deployment at: %s"
```

Comment:
Should we rather say "Task successfully created an Azure Resource Manager deployment, but the deployment failed. Please see more detailed Azure resource manager deployment logs at: "

Reply:
I've updated